src/commands/welcome.js

```diff
@@ -0,0 +1,17 @@
+const { makePipe, textContains } = require('./util')
+
+module.exports = makePipe(
+    textContains('thank', 'cheers', 'brilliant', 'you\'re a star', 'you have my gratitude'),
+    (text, message) => message.channel.send(chooseResponse())
+)
+
+const chooseResponse = () => responses[Math.floor(Math.random() * responses.length)]
+
+const responses = [
+    'You\'re welcome',
+    'Anytime!',
+    'No problem',
+    'Gratitude acknowledged',
+    '_nods_',
+    '_imperceptible nod_',
```

Comment:
Brilliant!

Reply:
I feel that `'...'` would also be quite in character.

Reply:
On it!